### PR TITLE
Update Hungarian language

### PR DESCRIPTION
**Pull request by AntoszHUN**
I updated the Hungarian language file with the file name language_hu.h.

    Translate newly added words.
    Fixed old translation errors.
    Correct incorrect letters.

Further fixes are coming soon.

Made by: AntoszHUN

### CODE REVIEW

**Comment by kakkas** [comment]
Köszi :)

File: Marlin/src/lcd/language/language_hu.h [hunk 1]
```diff
@@ -108,12 +111,15 @@ namespace Language_hu {
   PROGMEM Language_Str MSG_LASER_POWER                     = _UxGT("Lézer telj.");
   PROGMEM Language_Str MSG_SPINDLE_POWER                   = _UxGT("Orsó telj.");
   PROGMEM Language_Str MSG_LASER_TOGGLE                    = _UxGT("Lézer váltás");
+  PROGMEM Language_Str MSG_LASER_EVAC_TOGGLE               = _UxGT("Hütés váltás");
+  PROGMEM Language_Str MSG_LASER_ASSIST_TOGGLE             = _UxGT("Levegö segéd");
   PROGMEM Language_Str MSG_LASER_PULSE_MS                  = _UxGT("Impulzus teszt ms");
   PROGMEM Language_Str MSG_LASER_FIRE_PULSE                = _UxGT("Tüz impulzus");
+  PROGMEM Language_Str MSG_FLOWMETER_FAULT                 = _UxGT("Hütöfolyadék áramlási hiba");
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_hu.h]: This is 7 characters too long to be visible on most screens. Can this be reduced to <= 20 characters?

Comment (AntoszHUN) [reply Marlin/src/lcd/language/language_hu.h]: Thank you for the warning. I corrected.

Comment (thinkyhead) [reply Marlin/src/lcd/language/language_hu.h]: Thanks for the quick fix!

**Pull request merged**